examples/src/examples/AfiInteractiveOptimizerEngineExample.java

```diff
@@ -108,12 +112,28 @@ public static void main(String[] args) throws InterruptedException, JsonProcessi
       }
       
       ObjectMapper mapper = new ObjectMapper();
-      String json = mapper.writeValueAsString(tables);
-      System.out.println(json); // Prints the result in 2D table format.
+      // Prints the results in 2D table format.
+      for (TableData table : tables) {
+        List<Row> rows = table.getRows();
+        String json = mapper.writeValueAsString(rows);
+        System.out.println(json);
+      }
+      // Prints the metadata
+      for (TableData table : tables) {
+        if (table.getMetadata().size() > 0) System.out.println("Printing metadata...");
+        for (Map.Entry<String, List<Value>> rawMetadata : table.getRawMetadata().entrySet()) {
+          for (Value val : rawMetadata.getValue()) {
+            System.out.println("  " + rawMetadata.getKey() + ": " + StachUtilities.valueToString(val));
+          }
+        }
+      }
```

Comment:
Is it better to combine the for loop into one?
```
      // Prints the results in 2D table format.
      for (TableData table : tables) {
        List<Row> rows = table.getRows();
        String json = mapper.writeValueAsString(rows);
        System.out.println("Printing table data...");
        System.out.println(json);
      
        // Prints the metadata
        if (table.getMetadata().size() > 0) System.out.println("Printing metadata...");
        for (Map.Entry<String, List<Value>> rawMetadata : table.getRawMetadata().entrySet()) {
          for (Value val : rawMetadata.getValue()) {
            System.out.println("  " + rawMetadata.getKey() + ": " + StachUtilities.valueToString(val));
          }
        }
      }
```

Reply:
great suggestion! done here https://github.com/factset/analyticsapi-engines-java-sdk/pull/65/commits/817903b5016a8879c78d770c2e6a51dd4cfd1235